### PR TITLE
Fix typo in development guide

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

## Was the change discussed in an issue?
fixes #???

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -217,7 +217,7 @@ https://onsi.github.io/ginkgo/#focused-specs
 ### Race conditions
 
 It is not uncommon that during the execution of the integration tests, test failures occur.
-Although it's possible that this is due to the tests themselves, more often than not this occurs due to the the actual odo code is written.
+Although it's possible that this is due to the tests themselves, more often than not this occurs due to the actual odo code is written.
```
Comment (amitkrout) [inline docs/development.md]: ```...actual odo code...``` or ```...actual Odo code...``` ?

@kadel @cdrage Please confirm.
Otherwise looks good to me

Comment (kadel) [reply docs/development.md]: odo.  odo shouldn't be capitalized. That is the current standard that we should use.

**Comment by kadel** [comment]
> /lgtm
> 
> /hold cancel

sadly you still have to use github approve. Without 2 approves tide is not going to merge this automatically :-(

**Comment by mohammedzee1000** [comment]
> > /lgtm
> > /hold cancel
> 
> sadly you still have to use github approve. Without 2 approves tide is not going to merge this automatically :-(

Note: the reason tide can't merge is because it can't. Approving requests are required by github here. 
We can get around this by disabling requirement for 2 approving reviews,  but I'm not sure if we want to do this.

Instead hold off on lgtm or cancel of hold until 

1. 2 approves and no changes requested
1. All presubmits have passed including the Travis and circle Ci as they are required for merge.

**Pull request merged**